docs/providers/opensearch.md

```diff
@@ -0,0 +1,97 @@
+# Elasticsearch
+
+## Backend
+
+Using the `opensearch` backend class, you can query any metrics available in Opensearch to create an SLO.
```

Comment:
OpenSearch with a capital S, as used on [their website](https://opensearch.org/)? A global search/replace is probably the best option here.